tests/FSharpx.CSharpTests/ResultTests.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using Microsoft.FSharp.Core;
+using NUnit.Framework;
+
+namespace FSharpx.CSharpTests {
```

Comment:
It does not appear that this file was added to the test project. Also, might you consider adding tests for the CE and operators?

Reply:
New style csproj projects don't require file includes.